MediaAdminBundle/Manager/SaveMediaManager.php

```diff
@@ -135,7 +148,7 @@ public function saveMedia($media)
     protected function processMedia($media, $uploadedFile, $filename)
     {
         $media->setName($uploadedFile->getClientOriginalName());
-        $media->setMimeType($uploadedFile->getClientMimeType());
+        $media->setMimeType($this->getMimeType($filename));
```

Comment:
I tried it `$uploadedFile->getMimeType()` but it throwed a file not found exception, don't know why.
Moreover you already use the `finfo_file` method on the filename in the `MediaController` :
` if ($saveMediaManager->isFileAllowed($filename))`.